src/scripts/main.js

```
@@ -354,7 +354,62 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const getAge = (person) => {
+  if (!(person?.died || person?.born)) {
+    return '-';
+  }
 
-// write your code here
+  return person.died - person.born;
+};
+
+const getCentury = (person) => {
+  if (!person?.died) {
+    return '-';
+  }
+
+  return Math.ceil(person.died / 100);
+};
+
+const normalizeGender = (person) => {
+  const code = person?.sex;
+
+  if (code === 'm') {
+    return 'Male';
+  }
+
+  if (code === 'f') {
+    return 'Female';
+  }
+
+  return '-';
+};
+
+function filTable(table) {
+  const tBody = table.querySelector('tbody');
+
+  if (!tBody) {
+    return;
+  }
```

You get the table `tbody` and return early if none exists. The task asked to append created rows to the table element with class `dashboard`. If the HTML does not include a `<tbody>` you will silently do nothing. To be robust and adhere to the requirement, either: 1) create a `<tbody>` when missing (e.g. `const tBody = table.tBodies[0] || table.createTBody()`), or 2) append rows directly to the table using DOM methods. This ensures rows are inserted even when the tbody is absent.